uber/templates/regform.html

```diff
@@ -415,10 +422,19 @@
 <div class="form-group">
     <label for="email" class="col-sm-3 control-label">Email Address</label>
     <div class="col-sm-6">
-        <input type="email" type="email" name="email" id="email" value="{{ attendee.email }}" class="form-control" placeholder="Email Address">
+        <input type="email" name="email" id="email" value="{{ attendee.email }}" class="form-control" placeholder="Email Address">
```

Comment:
As has been pointed out to me in Slack, there's a `type=email` right before this. Whoops!